.github/workflows/build.yaml

```diff
@@ -49,9 +51,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: dart-lang/setup-dart@v1
+      - uses: subosito/flutter-action@v1
         with:
-          sdk: 2.12.0
+          flutter-version: '2.0.0'
```

Comment:
@dkrutskikh are you sure that the version should pinned to a lowest one? Maybe there is a way to always test it with the latest version?